mms/metrics_manager.py

```diff
@@ -57,39 +72,64 @@ def start(metrics_write_to, namespace, mutex):
         mutex: object
             Mutex to prevent double thread writing on same resource
         """
-        MetricsManager.metrics['error_metric'] = Metric('errors', mutex,
-                                                        namespace=namespace,
-                                                        aggregate_method='interval_sum',
-                                                        write_to=metrics_write_to)
-        MetricsManager.metrics['request_metric'] = Metric('requests', mutex,
-                                                          namespace=namespace,
-                                                          aggregate_method='interval_sum',
-                                                          write_to=metrics_write_to)
+        for model_name, model_class in models.items():
+            MetricsManager.metrics['500error_metric'] = Metric('500error_metric', mutex,
+                                                                model_name=model_name,
+                                                                unit=Unit['count'],
+                                                                aggregate_method='interval_sum',
+                                                                write_to=metrics_write_to)
+            MetricsManager.metrics['400error_metric'] = Metric('400error_metric', mutex,
+                                                                model_name=model_name,
+                                                                unit=Unit['count'],
+                                                                aggregate_method='interval_sum',
+                                                                write_to=metrics_write_to)
+            MetricsManager.metrics['request_metric'] = Metric('requests', mutex,
```

Comment:
Can't we use the same string key instead of having both "request_metric" and "requests"?
Same for all of the next metrics.

Reply:
changed